osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -2717,6 +2717,36 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/crescent/images/cre.svg"
       },
       "coingecko_id": "crescent-network"
+    },
+    {
+      "description": "The native token of Oraichain",
+      "denom_units": [
+        {
+          "denom": "ibc/161D7D62BAB3B9C39003334F1671208F43C06B643CC9EDBBE82B64793C857F1D",
+          "exponent": 0,
+          "aliases": [
+            "orai"
+          ]
+        },
+        {
+          "denom": "orai",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/161D7D62BAB3B9C39003334F1671208F43C06B643CC9EDBBE82B64793C857F1D",
+      "name": "Oraichain",
+      "display": "orai",
+      "symbol": "ORAI",
+      "ibc": {
+        "source_channel": "channel-13",
+        "dst_channel": "channel-216",
+        "source_denom": "orai"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/orai.png",
+        "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/orai.svg"
```

Comment:
```suggestion
        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.png",
        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai-white.svg"
```

Reply:
okay I will change to using these image logo urls

Reply:
No, I meant just have it point to that link.  Don't need to change the images.  We want the links in this repo to still point to images in the chain-registry repo.

Reply:
just fyi, the 'orai' images uploaded here are white, while the 'orai' images originally uploaded to the chain registry are black. This is why I also uploaded 'orai-white' images to the chain registry so either white or black could be referenced. It appears that the white images were intended, so I'd agree with the suggested change, which not only points to the chain registry as required, but also uses the white image variations.